kuma/static/js/wiki-edit-draft.js

```diff
@@ -0,0 +1,428 @@
+(function ($, win, doc) {
+    'use strict';
+
+    // feature test variable
+    var supportsLocalStorage = win.mdn.features.localStorage;
+    // some knowledge about the page we're dealing with
+    var $translateForm = $('#translate-document');
+    var $form = $translateForm.length ? $translateForm : $('#wiki-page-edit');
+    var isNewTranslation = location.search.indexOf('?tolocale') > -1 ? true : false;
+    // stuff we need for drafts
+    var DRAFT_NAME;
+    var draftTimeOutId;
+    var draftAutoSaveEnabled = true;
+    var draftWait = false;
+    var draftWaitTime = 500;
+    var $revisionInputs = $('[name=current_rev]'); // can't use ID because there's a duplicate >_<
```

Comment:
That would be nice. It broke things when I tried and it's an ancient problem so I figured it could be dealt with separately but if you want to pair on this we could try.